spec/controllers/admin/legislations_controller_spec.rb

```diff
@@ -2,7 +2,16 @@
 
 RSpec.describe Admin::LegislationsController, type: :controller do
   let(:admin) { create(:admin_user) }
+  let(:location) { create(:location) }
+
   let!(:legislation) { create(:legislation) }
+  let(:valid_attributes) {
+    attributes_for(:legislation).merge(
```

Comment:
I couldn't find a way to include location by default in what's returned by `attributes_for` :/ then I found your approach from other models and used it here as well.

Reply:
Yes, attributes_for are not working with associations, probably they have a good reason for that.